### DISCUSSION TITLE
Copilot limitation

### DISCUSSION THREAD

**Discussion by rafaltecza** [community/community#40361] [Copilot Conversations]
Hi,

I'm not sure, but I did some research on this case, and I'm a little surprised because this is a **critical unresolved** issue, and I've seen a lot of topics about it without any good answer. You cannot access multiple Copilot accounts on different PC's using same internet connection. It only works for the first person. By the way, there is no information about the error. Just after 0.5 seconds, the login box disappears. I switched from a WI-FI connection to a mobile network, and after I restarted IDE, it started to work again.

I am not able use the copilot for almost 2 months now.

**Comment by iamcjwilliams** [comment]
@rafaltecza Sorry for the trouble here, Can you provide a screenshot or log of the error you're receiving?

**Comment by davecheney** [comment]
@rafaltecza we do not limit the number of Copilot clients behind a single IP; technically each VSCode window holds its own copy of the copilot so multiple connections from a single computer behind a single IP are not unexpected. 

Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.
